Main/Controllers/UsersController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using LinkCollectionApp.Data;
+using LinkCollectionApp.Infrastructure.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LinkCollectionApp.Controllers
+{
+  [Authorize]
+  [Route("api/[controller]")]
+  [ApiController]
+  public class UsersController : ControllerBase
+  {
+    private readonly ApplicationDbContext _dbContext;
+    private readonly IUserContextProvider _userContextProvider;
+
+    public UsersController(IUserContextProvider userContextProvider, ApplicationDbContext dbContext)
+    {
+      _userContextProvider = userContextProvider;
+      _dbContext = dbContext;
+    }
+
+    [HttpGet]
+    public List<object> GetFriends()
```

Comment:
As far as i can tell this function does not return "friends" of the current user (btw what is the definition of a friend in this app? maby someone who shared his collection with us or someone who can edit/view our collection or both?). This function basicly returns all users except for current, not friends.

Reply:
There is no definition and we did not plan on doing this. I named it friends thinking sharing should be only to friends, but since we don't have time for everything, getting all users is fine for presentation purposes.
I should have put a TODO there, tho